features/other_a/abuse_report.feature

```diff
@@ -53,10 +53,11 @@ Feature: Filing an abuse report
     And I fill in "Your comment (required)" with "This is wrong"
     And I fill in "Link to the page you are reporting" with "http://www.archiveofourown.org/works"
     And I fill in "Your email (required)" with ""
+    And I select "Deutsch" from "abuse_report_language"
     And I press "Submit"
     And I should see "Email does not seem to be a valid address."
+    And the field labeled "abuse_report_language" should contain "Deutsch"
```

Comment:
This will pass without your fix because it's checking if "Deutsch" appears in the select, not if it's selected. You should use the step:

```
And "Deutsch" should be selected within "Select language (required)"
```